src/Commands/AdminCommands/SendtochannelCommand.php

```diff
@@ -33,7 +33,7 @@ class SendtochannelCommand extends AdminCommand
     /**
      * Conversation Object
      *
-     * @var Longman\TelegramBot\Conversation
+     * @var \Longman\TelegramBot\Conversation
```

Comment:
@MBoretto Did you deliberately leave all full namespaces as opposed to simplified ones?

PhpStorm automatically prefers simplified ones for me.

@akalongman @jacklul 
Asking you guys too, which standard shall we go by?
```php
@var \Longman\TelegramBot\Conversation
```
or
```php
@var Conversation
```
(this obviously implies that we have a `use Longman\TelegramBot\Conversation` at the top.